_layouts/default.html

```diff
@@ -9,6 +9,14 @@
         <!-- Javascript files -->
         <script src="/js/jquery.min.js"></script>
         <script src="/js/bootstrap.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/react/0.14.7/react.min.js"
```

Comment:
Yeah that's something I was wondering about. I'm not too familiar with the way that jekyll builds the pages so I'm not sure if we can add a headers section that appends headers to the default layout, or if we have to create an entirely new layout. For now I'll go in and create a new layout just for this page that includes all the necessary dependencies (ie momentjs and react) that are necessary for the starters page but aren't needed elsewhere.